css-two-column-layout/styles.css

```diff
@@ -0,0 +1,79 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: karla, sans-serif;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-one-half {
+  display: flex;
```

Comment:
If you are applying `display: flex` to a column class, that usually means that the `column` should contain a `row` class element. That is the purpose of the `row` class, to make flex containers